homeassistant/components/opower/strings.json

```diff
@@ -5,14 +5,16 @@
         "data": {
           "utility": "Utility name",
           "username": "[%key:common::config_flow::data::username%]",
-          "password": "[%key:common::config_flow::data::password%]"
+          "password": "[%key:common::config_flow::data::password%]",
+          "totp_secret": "TOTP Secret (optional)"
```

Comment:
It would be good to say, only for some utilities, see documentation. Otherwise people with an Exelon utility might be trying to enter a secret here. Same below.